<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPesquisaReservas</class>
 <widget class="QDialog" name="frmPesquisaReservas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciamento de Reservas</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLineEdit" name="txtPesquisa">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>0</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Termo de Pesquisa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdNome">
         <property name="text">
          <string>Nome</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdEmail">
         <property name="text">
          <string>E-mail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdFone">
         <property name="text">
          <string>Telefone</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btPesquisar">
         <property name="text">
          <string>Pesquisar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Lista de Reservas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblReservas">
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>9</number>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Data</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Nome</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fone</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>E-Mail</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Cidade</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>UF</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Prazo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>At</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btWhats">
             <property name="text">
              <string>WhastApp</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btEmail">
             <property name="text">
              <string>E-mail</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btHistorico">
             <property name="text">
              <string>Histórico E-mails</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btConfirma">
             <property name="text">
              <string>Confirmar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btOrcamento">
             <property name="text">
              <string>Orçamento</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btPrazo">
             <property name="text">
              <string>Prazo</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btExcluir">
             <property name="text">
              <string>Excluir</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btAtendida">
             <property name="text">
              <string>Atendida</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Itens da Reserva</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tblItens">
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <column>
          <property name="text">
           <string>id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Merc</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Clone</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Descrição</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Qtde</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Valor</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="txtQtde">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>33</height>
              </size>
             </property>
             <property name="inputMask">
              <string/>
             </property>
             <property name="maxLength">
              <number>32767</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbVariedade">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>33</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtValor">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>33</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="placeholderText">
              <string>0,00</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txtTotalItem">
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>33</height>
              </size>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="placeholderText">
              <string>0,00</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btAdicionar">
             <property name="text">
              <string>Adicionar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btAltera">
             <property name="text">
              <string>Alterar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btExcluir_item">
             <property name="text">
              <string>Excluir</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtPesquisa</tabstop>
  <tabstop>rdNome</tabstop>
  <tabstop>rdEmail</tabstop>
  <tabstop>rdFone</tabstop>
  <tabstop>btPesquisar</tabstop>
  <tabstop>tblReservas</tabstop>
  <tabstop>btWhats</tabstop>
  <tabstop>btEmail</tabstop>
  <tabstop>btHistorico</tabstop>
  <tabstop>btConfirma</tabstop>
  <tabstop>btOrcamento</tabstop>
  <tabstop>btPrazo</tabstop>
  <tabstop>btExcluir</tabstop>
  <tabstop>btAtendida</tabstop>
  <tabstop>tblItens</tabstop>
  <tabstop>txtQtde</tabstop>
  <tabstop>cmbVariedade</tabstop>
  <tabstop>txtValor</tabstop>
  <tabstop>txtTotalItem</tabstop>
  <tabstop>btAdicionar</tabstop>
  <tabstop>btAltera</tabstop>
  <tabstop>btExcluir_item</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
